core/iDDS/workflowprogress.py

```diff
@@ -56,6 +56,7 @@ def get_workflow_progress_data(request_params, **kwargs):
         ).reset_index()
         # fill NAN with 0 for N files
         workflows_pd.TOTAL_FILES.fillna(value=0, inplace=True)
+        workflows_items.R_NAME.fillna(value='', inplace=True)
```

Comment:
Hi @zhaoyuyoung , are you sure it is `workflow_items` and not `workflows_pd`? Or may be move this to line ~43?

Reply:
Thanks for the catch, I added to the wrong place in a rush when copy and paste from my yesterday's test. That line should be under line 43.

Reply:
Good, approving it now :)